source/documentation/reference/operational-processes.md

```diff
@@ -0,0 +1,178 @@
+
+#### Hours of support
+
+Our hours of providing support are 10am - 5pm. During this time we will work on support requests from teams and make sure someone is available to answer questions in Slack channel [`#ask-cloud-platform`](https://mojdt.slack.com/messages/C57UPMZLY/) and look at PR request notifications for the cloud-platform-environments repo on [`#cloud-platform-notify`](https://mojdt.slack.com/messages/CA5MDLM34/)
+
+Outside of these hours we will respond to [high priority](#prioritising-incidents) incidents as per our [on call](#our-on-call-process) process.
+
+#### Working hours
+
+A member of the support team will be available online from 9am and until 6pm. This team member can be working remotely. We are doing this so that we cover incident response for the time when the person on call is coming into the office or going home at the end of the day.
+
+This support team member should be ready to respond to any high priority incidents.
+
+#### The support sub team
+
+The support sub team will consist of 1-2 engineers. Each cloud platform team member will be on the sub team on a regular basis.
+
+The key activities for the day are:
+
+_On starting the day (9AM)_
+At least one engineer to:
+
+* Check the [`#low-priority-alarms`](https://mojdt.slack.com/messages/C8QR5FQRX/) and [`#high-priority-alarms`](https://mojdt.slack.com/messages/C8PF51AT0/) slack channels for any issues to investigate
+* Check the board for any user requests that have been raised and not assigned
+* Get a handover from the on call engineer about any issues out of hours
+* Read through [`#ask-cloud-platform`](https://mojdt.slack.com/messages/C57UPMZLY/) slack channel
+
+_During support hours (10AM - 5PM)_
+At least one engineer to:
+
+* Actively participate in [`#ask-cloud-platform`](https://mojdt.slack.com/messages/C57UPMZLY/) to field support requests, triage, prioritise and fix
+* Monitor [`#high-priority-alarms`](https://mojdt.slack.com/messages/C8PF51AT0/) and [`#low-priority-alarms`](https://mojdt.slack.com/messages/C8QR5FQRX/) for incidents, triage, prioritise and fix
+* Monitor [`#cloud-platform-notify`](https://mojdt.slack.com/messages/CA5MDLM34/) for cloud-platform-environments repo PR request notifications and cloud platform build notifications.
+* Monitor user support requests being added to the board and triage them for priority
+
+The whole support team to:
+
+* Manage incident process for incidents in progress
+* Work on user requests for larger work
+* Work on support backlog stories
+
+_Before ending the day (6PM)_
+At least one engineer to:
+
+* Handover information about any planned work or in progress high priority incidents to on call engineer
+
+#### `#ask-cloud-platform` slack channel
+
+The [`#ask-cloud-platform`](https://mojdt.slack.com/messages/C57UPMZLY/) channel is our main entry point for support. We encourage people to ask questions and report problems in this channel and we'll do the best we can to help.
```

Comment:
Can we also add a paragraph under this one along the lines of "The [`#ask-cloud-platform`](https://mojdt.slack.com/messages/C57UPMZLY/) channel is also the place for community discussion. Users can provide feedback of their experience using the Cloud Platform, as well as share their learning with other users of the platform.

Reply:
sentence added